templates/crypto/crypto.jinja2

```diff
@@ -0,0 +1,9 @@
+- test:
+    timeout:
+      minutes: 10
+    definitions:
+    - repository: https://github.com/linaro/test-definitions
```

Comment:
Please use the `kernelci` repository, which tracks the `linaro` upstream:
https://github.com/kernelci/test-definitions.git
with the `kernelci.org` branch.